src/Symfony/Component/Finder/Finder.php

```diff
@@ -169,14 +169,19 @@ public function date(string|array $dates): static
      *     $finder->name(['test.py', 'test.php'])
      *
      * @param string|string[] $patterns A pattern (a regexp, a glob, or a string) or an array of patterns
+     * @param bool            $overwriteExistingNames Whether to overwrite existing names or add them
      *
      * @return $this
      *
      * @see FilenameFilterIterator
      */
-    public function name(string|array $patterns): static
+    public function name(string|array $patterns, bool $overwriteExistingNames = false): static
```

Comment:
This is a BC break.
It needs to be handled like [here](https://github.com/symfony/symfony/pull/61548/files#diff-c11bc2eed1bcb41dd06ffc51051cd9f06afa699f56d55fdcb9e5de396550839d) and then properly introduced in 8.0 branch